src/status_im/contexts/communities/actions/addresses_for_permissions/view.cljs

```diff
@@ -249,18 +250,20 @@
         (fn []
           (if can-edit-addresses?
             (rf/dispatch
-             [:password-authentication/show
-              {:content (fn [] [password-authentication/view])}
-              {:label    (i18n/label :t/enter-password)
-               :on-press (fn [password]
-                           (rf/dispatch
-                            [:communities/edit-shared-addresses
-                             {:community-id id
-                              :password     password
-                              :addresses    addresses-to-reveal
-                              :on-success   (fn []
-                                              (rf/dispatch [:dismiss-modal :addresses-for-permissions])
-                                              (rf/dispatch [:hide-bottom-sheet]))}]))}])
+             [:standard-auth/authorize
+              {:auth-button-label (i18n/label :t/confirm-changes)
+               :on-auth-success   (fn [password]
+                                    (rf/dispatch
+                                     [:communities/edit-shared-addresses
+                                      {:community-id id
+                                       :password     (security/safe-unmask-data password)
```

Comment:
It's best to pass the password masked across events and only unmask it as it's passed to status-go. Masking prevents the password from accidentally being logged in the events.

Reply:
Done